configs/_base_/datasets/coco_detection.py

```diff
@@ -2,23 +2,29 @@
 dataset_type = 'CocoDataset'
 data_root = 'data/coco/'
 
-# file_client_args = dict(
+# Example to use different file client
+# Method 1: Infer from prefix (not support LMDB and Memcache)
+
+# data_root = 's3://openmmlab/datasets/detection/coco/'
+
+# Method 2: Use backend_args
+# backend_args = dict(
```

Comment:
We can make the comments in this file complete and clear, then update the same comments in other configs.